docs/modules/ROOT/pages/configuration-store.adoc

```diff
@@ -0,0 +1,172 @@
+= Universal Configuration Store
+
+== Overview
+The Universal Configuration Store is a tool that enables the various configurations for a project to be centrally defined and managed. 
+It then provides a standardized way of accessing them, allowing the environment specific configurations to be dynamically provided to 
+their respective resources within the project at runtime.
+
+=== Setup
+To setup the configuration store, one must first create properties files for storing the configurations. Each individual configuration is 
+represented as a property key value pair within the file. These files must then be made accessible to the configuration store kubernetes 
+resource for ingestion. See the applicable sections below for more details on defining configurations as properties, creating environment 
+specific property overrides, mounting the properties, and specifying the configuration store backend storage type.
+
+==== Defining Configuration Properties
+Properties represent key values pairs for storing the project configurations. The key values pairs are stored as string types using 
+the standard `.properties` file format. Each property then has an associated group name determined by the name of the file. Properties
+are then accessed using a combination of their group name and property name. Accessing the properties is covered in more detail in the 
+<<Usage>> section below.
+
+Here is an example of a base properties file named `messaging.properties`:
+[source,properties]
+----
+connector=smallrye-kafka
+topic=baseTopic
+----
+This file defines two properties, `connector` and `topic`, each with the group name `messaging`.
+
+===== Environment Specific Overrides
+In addition to the base properties file introduced above, one can optionally specify a secondary properties file that houses environment
+specific configurations. These configurations will then used to override their respective base configurations.
+
+Continuing with the example above, suppose one defines an additional properties file named `messaging.properties` like the following:
+[source,properties]
+----
+topic=ciTopic
+auth-token=auth-secret
+----
+
+Then when these two properties files are reconciled within the store, it results in a configuration equivalent to the following:
+[source,properties]
+----
+connector=smallrye-kafka
+topic=ciTopic
+auth-token=auth-secret
+----
+
+NOTE: The `topic` property is overwritten when the environment specific value `ciTopic`.
+
+==== Mounting Properties
+In order for the store to read the configurations, the properties files will need to be placed into a https://kubernetes.io/docs/concepts/storage/persistent-volumes/#lifecycle-of-a-volume-and-claimpersistent[Persistent Volume,role=external,window=_blank] (PV)
+mounted to the configuration store kubernetes resource. It is assumed a PV will meet most projects needs and is the most widely accepted 
```

Comment:
S: We've got a fine line to walk here between convenience/comprehensiveness in the antora docs, and confining chart-specific details to the chart documentation.  I think what we have is a pretty good split, but I think we could pull it up even a little higher here. For instance instead of getting into "Persistent Volumes" specifically, we can just say the high-level goal:  Attaching the properties to the local file system of the service through a mounted volume.   I don't have great specifics here, but maybe the key question is: Could someone with very little Kubernetes/Helm knowledge read the paragraph and get good understanding of _what's_ happening and why (even if they don't fully understand the "how").

Reply:
Brought up to higher level by removing kubernetes specific wording.